### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by kofiarkoh**
Adds tests for 
- login
- register
- logout

### CODE REVIEW

**Comment by lingjunliu** [comment]
umm you're trying to merge this branch to "main" branch...
I will close this pull request

**Comment by kofiarkoh** [comment]
no need to close, i could just update the branches

File: requirements.txt [hunk 1]
```diff
@@ -1,16 +1,24 @@
 amqp==5.2.0
+amqp==5.2.0
 bcrypt==3.2.0
+beautifulsoup4==4.12.3
+billiard==4.2.1
 billiard==4.2.1
 blinker==1.4
 Bottleneck==1.3.7
 celery==5.4.0
+celery==5.4.0
```
Comment (lingjunliu) [inline requirements.txt]: repeated packages

File: requirements.txt [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
 certifi==2024.8.30
 cffi==1.14.6
 charset-normalizer==3.4.0
 click==8.1.7
 click-didyoumean==0.3.1
 click-plugins==1.1.1
 click-repl==0.3.0
+click==8.1.7
```
Comment (lingjunliu) [inline requirements.txt:18]: repeated as well. recommend to read through this file and check repetition

File: requirements.txt [hunk 3]
```diff
@@ -1,16 +1,24 @@
...
+click-didyoumean==0.3.1
+click-plugins==1.1.1
+click-repl==0.3.0
```
Comment (lingjunliu) [inline requirements.txt]: repeated again. recommend to read through this file to check repetition

File: requirements.txt (hunk 1 above)
Comment (kofiarkoh) [reply requirements.txt]: updated

File: requirements.txt (hunk 3 above)
Comment (kofiarkoh) [reply requirements.txt]: fixed

File: README.md [hunk 4]
```diff
@@ -243,6 +243,8 @@ https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows
 
 ```
 NOTE!!:
+Make sure you have redis install on your local machine for background tasks with celery to work. see https://redis.io/docs/latest/operate/oss_and_stack/install/
+
```
Comment (lingjunliu) [inline README.md:247]: Do we still need redis?

Comment (lingjunliu) [reply README.md:247]: This can be resolved later. I'll merge first

**Pull request merged**